topics/assembly/tutorials/assembly-with-preprocessing/slides.html

```diff
@@ -0,0 +1,322 @@
+---
+layout: tutorial_slides
+logo: "GTN"
+class: enlarge120
+
+title: "Unicycler assembly of SARS-CoV-2 genome with preprocessing to remove human genome reads"
+zenodo_link: "https://zenodo.org/record/3732359"
+questions:
+  - "How can a genome of interest be assembled against a background of contaminating reads from other genomes?"
+  - "How can sequencing data used to obtain an assembled genome?"
+objectives:
+  - "Know basic characteristics of SARS-CoV-2"
+  - "Understand Nanopore and Illumina technologies"
+  - "Detect and remove human reads from SARS-CoV-2 sequencing reads"
+  - "Know main _de novo_ genome assembly algorithms"
+  - "Perform a hybrid _de novo_ genome assembly "
+time_estimation: "4h"
+key_points:
+  - "Certain types of NGS samples can be heavily contaminated with sequences from other genomes."
+  - "Reads from known/expected contaminating sources can be identified by mapping to the respective genomes."
+  - "The different characteristics of Illumina and Nanopore sequencing technologies require processing by different tools."
+  - "Hybrid genome assembly allows to obtain high quality genome sequences."
+contributors:
```

Comment:
sorry for the note after, but, could you please tag this as covid19? So it shows up here https://training.galaxyproject.org/training-material/search?query=covid19